routes/extra_data_utils.go

```diff
@@ -0,0 +1,64 @@
+package routes
+
+import (
+	"encoding/hex"
+	"fmt"
+	"github.com/deso-protocol/core/lib"
+	"strconv"
+)
+
+type ExtraDataDecoder func([]byte) string
+
+var ExtraDataKeyToDecoder = map[string]ExtraDataDecoder{
+	lib.RepostedPostHash:  hex.EncodeToString,
+	lib.IsQuotedRepostKey: Decode64BitIntString,
```

Comment:
IsQuotedRepostKey maps to a value that represents a boolean

Reply:
Yeah, it's represented as a byte with a value of {0, 1}. The functionality here that decode the value works the same for any length integer up to a length of 8 bytes. It still gives us a decoded value of {0, 1}. I'll create a new function for a more appropriate name for it to avoid confusion though